src/JWTVerifier.php

```diff
@@ -138,12 +138,16 @@ public function __construct(array $config, JWKFetcher $jwkFetcher = null)
         }
 
         // Only store the client_secret if this is an HS256 token.
+        // This only runs if $this->supported_algs is ['HS256'] or ['HS256', 'RS256'].
```

Comment:
In what case do we want "supported algs" to allow having two values?

Reply:
We don't. We're forcing one or the other with this change.